reference/supervisor.md

```diff
@@ -1934,8 +1934,8 @@ wb_supervisor_simulation_reset()
 
 The `wb_supervisor_simulation_reset` function sends a request to the simulator process, asking it to reset the simulation immediately.
 The reset process is explained in detail in the [User Guide](https://www.cyberbotics.com/doc/guide/the-user-interface#file-menu).
-As a result of the reset, the supervisor process and all the robot processes are terminated and restarted.
-You may wish to save some data in a file from your supervisor program in order to reload it when the supervisor controller restarts.
+As a result of the reset, all the supervisor and robot controller processes are terminated and restarted.
+You may wish to save some data in a file from your supervisor and robot controller programs in order to reload them when they restart.
```

Comment:
Data is always singular. Therefore we should say "reload it" instead of "reload them".

Reply:
Are you sure, for me it was always plural. From the oxford dictionary (https://en.oxforddictionaries.com/definition/data):
> In Latin, data is the plural of datum and, historically and in specialized scientific fields, it is also treated as a plural in English, taking a plural verb ... In modern non-scientific use, however, it is generally not treated as a plural. 

I think we can consider this doc as 'scientific' no?